packages/kinto/import-solen.py

```diff
@@ -112,39 +102,70 @@ def importPeriodeDeReference(self):
+        self.importField("Code NAF", "informationsEntreprise/codeNaf")  # attention format
 
     def importUES(self):
-        self.importField("nom_UES", "informationsEntreprise.nomUES")
-        self.importField("nom_ets_UES", "informationsEntreprise.nomsEntreprisesUES")
-        self.importField("SIREN_UES", "informationsEntreprise.sirenUES")
-        self.importField("Code NAF de cette entreprise", "informationsEntreprise.codeNAF")  # attention format
-        self.importIntField("Nb_ets_UES", "informationsEntreprise.nombresEntreprises")
+        self.importField("nom_UES", "informationsEntreprise/nomUES")
+        self.importField("nom_ets_UES", "informationsEntreprise/nomsEntreprisesUES")
+        self.importField("SIREN_UES", "informationsEntreprise/sirenUES")
+        self.importField("Code NAF de cette entreprise", "informationsEntreprise/codeNAF")  # attention format
+        # FIXME: bug avec le champ "Nb_ets_UES" dans le nouveau format d'export fourni par Ponn,
+        # contenant des dates au lieu de l'entier escompté
+        # self.importIntField("Nb_ets_UES", "informationsEntreprise/nombresEntreprises")
 
     def importNiveauResultat(self):
         # Niveau de résultat de l'entreprise ou de l'UES
-        self.importField("date_publ_niv > Valeur date", "declaration.datePublication")
-        self.importField("site_internet_publ", "declaration.lienPublication")
+        self.importField("date_publ_niv > Valeur date", "declaration/datePublication")
+        self.importField("site_internet_publ", "declaration/lienPublication")
+
+    def setValeursTranche(self, column, path, fieldName):
+        columns = [column + " > -30", column + " > 30-39", column + " > 40-49", column + " > 50+"]
+        values = [self.get(col) for col in columns]
+        tranches = [None, None, None, None]
+        for index, value in enumerate(values):
+            tranches[index] = {"trancheAge": index}
+            if value is not None:
+                tranches[index][fieldName] = atof(value)
+        self.set(path, {"tranchesAges": tranches})
+
+    def setValeursTranches(self, columns, path, fieldName, max=4):
+        for index, column in enumerate(columns):
+            if index + 1 > max:
+                break
+            self.setValeursTranche(column, "{0}/{1}".format(path, index), fieldName)
+
+    def importTranchesCsp(self):
+        self.set("indicateurUn/remunerationAnnuelle", [])
+        self.setValeursTranches(["Ou", "Em", "TAM", "IC"], "indicateurUn/remunerationAnnuelle", "ecartTauxRemuneration", max=4)
+
+    def importTranchesCoefficients(self):
+        try:
+            max = int(self.get("nb_coef_niv"))
+        except Exception as err:
+            raise RuntimeError("Valeur nb_coef_niv non renseignée, indispensable pour une déclaratio par niveaux de coéfficients")
+        self.set("indicateurUn/remunerationAnnuelle", [])
+        niveaux = ["niv{:02d}".format(niv) for niv in range(1, 51)]
```

Comment:
changer en `niveaux = ["niv{:02d}".format(niv) for niv in range(1, max)]` ?
Du coup plus besoin d'avoir un `max` dans `setValeursTranches`